charts/sysdig-deploy/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 ## Change Log
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
+
+## v1.4.6
+### Minor changes
+* agent:
+    * Added unit tests to check local overrides
+* node-analyzer:
+    * Added unit tests to check local overrides
+* kspm-collector:
+    * Added unit tests to check local overrides
+* rapid-response:
+    * Added unit tests to check local overrides
```

Comment:
nit: similar nit as to the above. maybe change to something like `Added unit tests to check local overrides of global values`?